release/charts/charts.go

```diff
@@ -45,12 +83,12 @@ func runChartsBuild(chartsRepoPath string, args ...string) ([]byte, error) {
 	cmd := exec.Command(bin, args...)
 	output, err := cmd.CombinedOutput()
 	if err != nil {
-		return nil, err
+		return output, err
```

Comment:
Why are we returning both? This is an anti pattern in go.

Reply:
I decided to return it in this line, 
because depending on the error, we might want to inspect the output to see what went wrong. 

Is this ok?

Reply:
You probably should append the output to the error message and not return it as a separate variable

Reply:
I think I understood it now, please take a look @tashima42